### ISSUE TITLE
no-extend-native does not catch extensions by Object.defineProperties

### ISSUE THREAD

**Issue by woollybogger** [eslint/eslint#4180]
Using `/* no-extend-native: 2 */` does catch this:

```js
Object.defineProperty(Object.prototype, 'lint', {value: 2});
```

but not this:

```js
Object.defineProperties(Object.prototype, {
  lint: {value: 2}
});
```

Using ESLint v1.7.1

**Comment by eslintbot** [comment]
Thanks for the issue! If you're reporting a bug, please be sure to include:

1. The version of ESLint you are using (run `eslint -v`)
2. What you did (the source code and ESLint configuration)
3. The actual ESLint output complete with numbers
4. What you expected to happen instead

Requesting a new rule? Please see [Proposing a New Rule](http://eslint.org/docs/developer-guide/contributing/new-rules) for instructions.
